controls/bsi_app_4_4.yml

```diff
@@ -184,8 +184,9 @@ controls:
       start pods via automation software, this SHOULD be done for each group through separate
       processes that only have the rights necessary for the respective user group.
     notes: >-
-      TBD
-    status: pending
+      This control needs to be adressed on an organizational level. All service accounts used by 
+      automation software need to adhere to the principal of least privilege.

```

Comment:
```suggestion
      automation software need to adhere to the principle of least privilege.

```

Reply:
Thanks for review, I will add this to the previous commit to keep commit history clear.